skymp5-server/cpp/server_guest_lib/CropHealthRegeneration.cpp

```diff
@@ -1,12 +1,27 @@
 #include "CropHealthRegeneration.h"
+#include "GetBaseActorValues.h"
+#include "MpActor.h"
+
+float PercentToFloat(float percent)
```

Comment:
Functions like these should be placed in private namespaces to avoid linker issues in future